spec/controllers/health_spec.rb

```diff
@@ -0,0 +1,105 @@
+require 'spec_helper'
+require 'rake'
+
+describe HealthController, type: :controller do
+  describe "Hitting the health endpoint of Earthdata Search" do
+    before :all do
+      @delayed_job = DelayedJob.new
+      @delayed_job.run_at = Time.now
+
+      @retrieval = Retrieval.new
+      @retrieval.created_at = Time.now - 1.minute
+    end
+
+    before :each do
+      Dir.glob(Rails.root.join('tmp', "data_load_*")).each { |f| File.delete(f) }
+      Dir.glob(Rails.root.join('tmp', "colormaps_load_*")).each { |f| File.delete(f) }
+
+      FileUtils.touch Rails.root.join('tmp', "data_load_#{Time.now.to_i}")
+      FileUtils.touch Rails.root.join('tmp', "colormaps_load_#{Time.now.to_i}")
+    end
+
+    after :each do
+      Dir.glob(Rails.root.join('tmp', "data_load_*")).each { |f| File.delete(f) }
+      Dir.glob(Rails.root.join('tmp', "colormaps_load_*")).each { |f| File.delete(f) }
+    end
+
+    context "when everything is up" do
+      it "should return a json response indicating the system is ok" do
```

Comment:
Language for specs shouldn't use "should".  Just, "when everything is up, it returns a json response indicating the system is ok" (similar for specs below).